prov/efa/src/efa_static_headers.h

```diff
@@ -0,0 +1,19 @@
+#ifndef EFA_STATIC_HEADERS_H
+#define EFA_STATIC_HEADERS_H
+#include <stdlib.h>
+#include <stdint.h>
+#include "rxr.h"
+#include "efa.h"
+#ifdef EFA_UNIT_TEST
+
+/**
+ * This file contains headers for static functions that are converted
+ * to non static functions when compiled under unit testing.
+ *
+ */
+
+int rxr_info_to_rxr(uint32_t version, const struct fi_info *core_info,
```

Comment:
Instead of testing `rxr_info_to_rxr`, I think `rxr_getinfo`  is a better target, which is the entry point of application, and  the input and output is well defined. (or even `rxr_prov->get_info`, if testing a struct is possible)